tests/test_sap.py

```diff
@@ -401,3 +402,23 @@ def test_generate_sap_control(sap_data_file):
     assert sap_control[72:92] == "00000000000000136704"
     assert sap_control[92:112] == "00100100000000000000"
     assert len(sap_control.encode("utf-8")) == 113
+
+
+def test_generate_next_sap_sequence_number(mocked_ssm):
+    ssm = SSM()
+    assert (
+        ssm.get_parameter_value("/test/example/SAP_SEQUENCE")
+        == "1001,20210722000000,ser"
+    )
+    new_sap_sequence = sap.generate_next_sap_sequence_number()
+    assert new_sap_sequence == "1002"
+
+
+def test_update_sap_sequence(mocked_ssm):
+    ssm = SSM()
```

Comment:
This should probably also include the same assertion as in the test above to demonstrate that the parameter was something, and then was updated to something else.